features/not_crashing_behat_with_cross_container_extension.feature

```diff
@@ -1,4 +1,7 @@
-Feature: Not crashing Behat
+Feature: Useage of ServiceContainerExtension together with CrossContainerExtension
+    In order to declare own services and parameters inside Behat and other containers
```

Comment:
In order to reference cross-container services and parameters inside Behat container